### DISCUSSION TITLE
Palworld is doing crazy numbers for us right now

### DISCUSSION THREAD

**Discussion by Buckminsterfullerene02** [UE4SS-RE/RE-UE4SS#352] [General]
Just thought I'd leave this here for history's sake. Always interesting to look back on stuff like this (especially when github traffic graph history is limited to 14 days).

Palworld, peaking at 2.1 million concurrent players on Steam (and 950k every day at the moment), is kind of blowing up UE4SS. Just look at these numbers! Before Palworld, even after UEVR released and referenced UE4SS as the more suitable tool than itself for actual modding, the repo was seeing roughly 1,000 views per day. 

![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/84156063/b5a5ec16-ce40-4e60-a5f8-71ee83722da5)

We then peaked at 71,694 views, 19,500 of those being from unique users, on 5th Feb when v3.0.0 was released. The drop of the graph is today's date, which for me at time of writing is only halfway through, so it makes sense that it's much lower right now. Though it's highly likely the general trend from v3.0.0 release will be downwards due to the rapidly declining popularity of Palworld. It'll be interesting to see how it develops over time though.

**Comment by UE4SS** [comment]
This is cool and all, but Palworld is not free, at least not as far as I can tell.

**Reply by Buckminsterfullerene02** [reply]
Wait what I swear I did not spend £22 on this game... I thought it was free! Oh well, apologies for that mistake

**Comment by Buckminsterfullerene02** [comment]
A month on, still getting around 10k views or 5k unique views every day (and we hit 800 stars woo!)
![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/84156063/98993c0b-1f41-47fc-81f7-be808b4e1c82)
